lib/std/os/linux/syscalls.zig

```diff
@@ -0,0 +1,3505 @@
+// This file is automatically generated.
+// See tools/generate_linux_syscalls.zig for more info.
+pub const SYS = switch (@import("builtin").cpu.arch) {
+    .i386 => SysX86,
+    .x86_64 => SysX64,
+    .aarch64 => SysARM64,
+    .arm, .thumb => SysARM,
+    .riscv64 => SysRV64,
+    .sparc64 => SysSparc,
+    .mips, .mipsel => SysMips,
+    .powerpc => SysPPC,
+    .powerpc64, .powerpc64le => SysPPC64,
+    else => @compileError("unsupported architecture"),
+};
+
+pub const SysX86 = enum(usize) {
```

Comment:
The fully qualified namespace to this declaration is `std.os.linux.syscalls.SysX86`. I suggest to change it so that it becomes `std.os.linux.syscalls.X86`:

```suggestion
pub const X86 = enum(usize) {
```

(same with the other architectures in this file)